dev-python/minimock/minimock-1.3.0.ebuild

```diff
@@ -3,12 +3,11 @@
 
```

Comment:
Well, interestingly enough, no coverage try is attempted on my clean container - so I don't understand how it happens.
Just in case, added a `sed` to clean every `cov` mention.

Could you retest if it helps on your machine?